src/mds/Mantle.h

```diff
@@ -0,0 +1,34 @@
+#include <lua.hpp>
```

Comment:
please add #ifdef header guards and the usual modelines+copyright header (copy paste and put your name in).  Same in Mantle.cc